src/main/kotlin/com/codecraft/ai/api/engine/ForFutureDevelopment.kt

```diff
@@ -0,0 +1,28 @@
+package com.codecraft.ai.api.engine
```

Comment:
do we really need this much commented code?

Reply:
Ok i will remove it